db/migrate_cards/data/cards/metric+topic_count+*type_plu_right+*structure

```diff
@@ -0,0 +1,4 @@
+{ 
+  "referred_to_by": "_l+topics",
+  "return" : "count"
+}
```

Comment:
question: do we have a pattern of creating count cards like this?

another approach would be to make it so that (metric)+topic returned the topic search; then you could use (metric)+topic+*count to return the count.